### DISCUSSION TITLE
Lazy loading by multiple keys

### DISCUSSION THREAD

**Discussion by sQVe** [wbthomason/packer.nvim#422] [Q&A] [answered]
Hey 👋🏼,

I've been trying to setup a couple of plugins to lazy load via the `keys` options. These plugins have multiple keybindings that spans over multiple modes. An example would be:

```lua
use {'tversteeg/registers.nvim', keys = {{'n', '"'}, {'i', 'ctrl-r'}}} -- Interactive registers.
```

The ☝🏼 works but only for the <kbd>"</kbd> under normal mode. When in insert mode <kbd>Ctrl + R</kbd> does not load `registers.nvim`. What am I doing wrong? 🤔 🙏🏼

**Comment by wbthomason** [comment, marked as answer]
Hi! Looks like you have an error in your keymap. In Vim/Neovim, to make a map with the ctrl key, you need to use the notation `<c-{the character you want to map}>`. So, in other words, your map should be `<c-r>`. The following works for me as expected:
`use {'tversteeg/registers.nvim', keys = {{'n', '"'}, {'i', '<c-r>'}}}`

Hope this helps!

**Comment by sQVe** [comment]
Awesome @wbthomason. That did the trick. I searched high and low but my Google-fu failed me. Also tried all the options I could think of but missed this one it seems.
